src/microbe_stage/editor/PatchMapNode.cs

```diff
@@ -9,21 +9,37 @@ public class PatchMapNode : MarginContainer
     [Export]
     public NodePath IconPath = null!;
 
+    // Selected patch node
```

Comment:
XML comments on parts of code, please. There's a ton of these in this file so I'll only comment once.